client/src/pages/teds/charts/PrctIpbesPublicationsByCountry.tsx

```diff
@@ -0,0 +1,60 @@
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import buildSeries from './utils/UseSeries.js';
```

Comment:
Best practices :
* Leave a blank line between imports from libs and imports from local files.
* Sort import by libs names or relative paths
* Leave a unique blank line between last import and class name